apps/new/bos.config.json

```diff
@@ -1,10 +1,12 @@
 {
   "account": "builddao.near",
   "aliases": ["../../aliases.mainnet.json"],
+  "index": "builddao.near/widget/Index",
```

Comment:
we can routes for all main index pages then?
like feed, projects.

Reply:
We could, in fact the entire "routes" config could be handled by a bos.config.json (or even just by folder structure alone). Is definitely something to explore (although bos-workspace doesn't yet support it)